nx-dev/nx-dev/pages/packages/rspack/executors/index.tsx

```diff
@@ -5,7 +5,7 @@ import { ProcessedPackageMetadata } from '@nx/nx-dev/models-package';
 import { DocumentationHeader, SidebarContainer } from '@nx/nx-dev/ui-common';
 import { useRouter } from 'next/router';
 import { useEffect, useRef } from 'react';
-import { PackageSchemaSubList } from '../../../../../feature-package-schema-viewer/src/lib/package-schema-sub-list';
+import { PackageSchemaSubList } from '@nx/nx-dev/feature-package-schema-viewer/src/lib/package-schema-sub-list';
```

Comment:
followup: @isaacplmann @bcabanes 

We should expose this as public api or another library from `feature-package-schema-viewer` since it is being used in other places.